WordPress/src/main/res/anim/prepublishing_fragment_fade_in.xml

```diff
@@ -1,10 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <set xmlns:android="http://schemas.android.com/apk/res/android"
     android:fillAfter="true" >
-
     <alpha
+        android:duration="@android:integer/config_mediumAnimTime"
+        android:interpolator="@android:anim/decelerate_interpolator"
         android:fromAlpha="0"
-        android:toAlpha="1"
-        android:startOffset="200"
-        android:duration="200" />
+        android:toAlpha="1" />
+<!--<alpha-->
```

Comment:
Is this change intentional - if it is, we might want to remove the commented out code. Wdyt?

Reply:
In 89a9945